websites/O/Oneanime/presence.ts

```diff
@@ -0,0 +1,108 @@
+const presence = new Presence({ clientId: "1028640811925114942" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video = {
+	current: 0,
+	duration: 0,
+	paused: true,
+};
+
+presence.on(
+	"iFrameData",
+	(data: { current: number; duration: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
```

Comment:
Imgur links are preferred over Discord asset names in `largeImageKey` and `smallImageKey`.